puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman_infra.groovy

```diff
@@ -1,6 +1,4 @@
-void git_clone_foreman_infra(args) {
-    target_dir = args.target_dir ?: ''
-
+void git_clone_foreman_infra(target_dir = '') {
```

Comment:
This signature shouldn't need changing.

Reply:
@ehelms  [here](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foremanRelease.groovy#L33) it is failing at this point at the moment. It doesn't seem to recognize that args.target_dir should exist and fails out. Is there another way to not have it do that?

[This](http://ci.theforeman.org/job/foreman-nightly-release/326/) is an example failure

Reply:
Oh maybe we need to set a default value for args, something like `args = {}` in the parameters.